UnityProject/Assets/Scripts/Inventory/ItemStorage.cs

```diff
@@ -16,8 +16,7 @@
 /// Note that items stored in an ItemStorage can themselves have ItemStorage (for example, storing a backpack
 /// in a player's inventory)!
 /// </summary>
-public class ItemStorage : MonoBehaviour, IServerLifecycle, IServerInventoryMove, IClientInventoryMove,
-	IClientDespawn
+public class ItemStorage : MonoBehaviour, IServerLifecycle, IServerInventoryMove, IClientInventoryMove,	IClientDespawn
```

Comment:
this could probably be split up over multiple lines at this point, otherwise I would remove the extra tab between `IClientInventoryMove` and `IClientDespawn`